src/lib/components/artwork_grids/artwork.tsx

```diff
@@ -59,10 +59,15 @@ class Artwork extends React.Component<RelayProps, any> {
   saleMessage() {
     const artwork = this.props.artwork
     if (artwork.is_in_auction) {
+      const numberOfBids = artwork.sale_artwork.bidder_positions_count
```

Comment:
Not a big deal (at all), but could use nested destructuring to dry some of this up:

```javascript
const { 
  sale_artwork: {
    bidder_position_count,
    opening_bid,
    ... 
  }
} = this.props 
```